napari/components/interaction_box.py

```diff
@@ -35,18 +37,18 @@ class InteractionBox(EventedModel):
         Bool indicating whether the interaction box allows creating a new selection by dragging outside an existing interaction_box.
     """
 
-    points: Array[float, (-1, 2)] = None
+    points: Optional[Array[float, (-1, 2)]] = None
     show: bool = False
```

Comment:
One benefit of this PR is that now we are forced to use the correct annotation in places like here :P